doc/python/recipe.rst

```diff
@@ -18,11 +18,23 @@ Recipe
     Describe a model by describing the cells and network, without any information about how the model is to be represented or executed.
 
     All recipes derive from this abstract base class.
-
+    
     Recipes provide a cell-centric interface for describing a model.
     This means that model properties, such as connections, are queried using the global identifier ``gid`` of a cell.
     In the description below, the term ``gid`` is used as shorthand for the cell with global identifier.
 
+    **Required Constructor**
+
+    The constructor must be implemented and call the base class constructor, as at the moment there is no way
```

Comment:
This is -- incidentally -- endemic to Python.
```
In [1]: class A:
   ...:     def __init__(self):
   ...:         print("A")
In [2]: class B(A):
   ...:     def __init__(self):
   ...:         print("B")
In [3]: a = A()
A
In [4]: b = B()
B
In [5]: class C(A):
   ...:     def __init__(self):
   ...:         super().__init__() # same a A.__init__() but more general
   ...:         print("C")
In [8]: c = C()
A
C
```

Reply:
I know, and it's unpleasant...